app/domain/authentication/authn_jwt/fetch_mandatory_claims_to_validate.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module Authentication
+  module AuthnJwt
+    OPTIONAL_CLAIMS = [ISS_CLAIM_NAME, EXP_CLAIM_NAME, NBF_CLAIM_NAME, IAT_CLAIM_NAME].freeze
+
+    # FetchMandatoryClaimsToValidate command class is responsible to return a list of mandatory JWT standard claims to
+    # validate.
+    FetchMandatoryClaimsToValidate ||= CommandClass.new(
+      dependencies: {
+        fetch_issuer_value: ::Authentication::AuthnJwt::FetchIssuerValue.new,
+        logger: Rails.logger
+      },
+      inputs: %i[authentication_parameters]
+    ) do
+
+      def call
+        @logger.debug(LogMessages::Authentication::AuthnJwt::FetchingJwtMandatoryClaimsToValidate.new)
+        fetch_mandatory_claims_to_validate
+        @logger.debug(LogMessages::Authentication::AuthnJwt::FetchedJwtMandatoryClaimsToValidate.new)
+
+        @mandatory_claims_to_validate_list
```

Comment:
i think that `list` is redundant. you can change it to `@mandatory_claims_to_validate`